src/main/java/com/yahoo/identity/services/storage/sql/AccountMapper.java

```diff
@@ -3,9 +3,12 @@
 import org.apache.ibatis.annotations.Param;
 
 public interface AccountMapper {
+
     int insertAccount(@Param("account") AccountModel account);
 
     AccountModel getAccount(@Param("username") String username);
 
     int updateAccount(@Param("account") AccountModel account);
+
+    int verifyUsername(@Param("username") String username);
```

Comment:
I think we don't need this.
We can just insert it. If unique constraint is violated, we can do corresponding handling.

Reply:
Removed.